src/Singularizer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Pokemon;
+
+use Doctrine\Common\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+
+class Singularizer
+{
+    public static function singularize(string $word): string
```

Comment:
This will build a new Inflector every time it's called, quite inefficient unfortunately.
Is there really a need to create an entire class for just this?

Reply:
It has been helpful to have an extra class for this so the compatibility has not to be implemented all the time there is the need of a singularize. But I am with you. I should cache that instance.

FYI: I wouldn't have done it anyway when I didn't want to use it in a project that would support the new inflector 😁

Reply:
I can live with it being it's own class, I don't think it should live in the base Pokemon namespace though. I think it should be put into a directory like `Common`, `Wrappers`, `Utilities` or `Helpers` and given the appropriate namespace. Depending on the directory/namespace, I'd also maybe change the class name to match `Inflector` (this is less important to me though).

Reply:
I think Common is a good namespace. I can amend.